ecolex/context_preprocessors.py

```diff
@@ -0,0 +1,6 @@
+def set_iframe(request):
+    if 'HTTP_REFERER' in request.META:
```

Comment:
Probably a bad idea, referrer is not only sent for iframes, but also when clicking on a normal link to the other website

Reply:
I suggest to implement an explicit request parameter, "&headless=true" instead